rclcpp_lifecycle/src/lifecycle_node.cpp

```diff
@@ -74,21 +74,22 @@ LifecycleNode::LifecycleNode(
   node_timers_(new rclcpp::node_interfaces::NodeTimers(node_base_.get())),
   node_topics_(new rclcpp::node_interfaces::NodeTopics(node_base_.get())),
   node_services_(new rclcpp::node_interfaces::NodeServices(node_base_.get())),
+  node_clock_(new rclcpp::node_interfaces::NodeClock(
+      node_base_,
+      node_topics_,
+      node_graph_,
+      node_services_,
+      node_logging_
+    )),  
```

Comment:
This line has trailing white spaces. @tfoote Please run the unit tests before merging PRs.

Reply:
Fixed in 33f1e1776c63a37623c0a1f006b4951e4522f071.

Reply:
Thanks @dirk-thomas I'm not sure how that passed. I ran CI here: https://github.com/ros2/rcl_interfaces/pull/51#issuecomment-444257583